src/nodes.coffee

```diff
@@ -535,6 +538,17 @@ exports.NaNLiteral = class NaNLiteral extends NumberLiteral
 
 exports.StringLiteral = class StringLiteral extends Literal
 
+  compileNode: (o) ->
+    res = if @csx then [@makeCode @unquote yes] else super()
+
+  unquote: (literal) ->
+    unquoted = @value[1...-1]
+    if literal
+      unquoted.replace /\\n/g, '\n'
+        .replace /\\"/g, '"'
```

Comment:
Dealing with backslashes is never this simple. What if the backslash is already escaped? `\\"`